### DISCUSSION TITLE
Example app for Dynamically Loaded Components returns null

### DISCUSSION THREAD

**Discussion by AyushExel** [idom-team/idom#1105] [Problem]
I'm following the example here https://reactpy.dev/docs/guides/escape-hatches/javascript-components.html#dynamically-loaded-components
And it works on the browser when clicking result button but on running it on my system, i get `null` page.
<img width="811" alt="Screenshot 2023-07-15 at 8 20 12 PM" src="https://github.com/reactive-python/reactpy/assets/15766192/28fefde5-729c-4fea-a354-173e3b5983b3">

**Comment by rmorshea** [comment]
I was able to reproduce what appears to be a similar issue: https://github.com/reactive-python/reactpy/issues/1106

Can you comment there with any logs you see in your browser's dev console?
